source/404.md

```diff
@@ -5,3 +5,12 @@ containerClass: not-found
 ---
 
 # 404 Page not Found
+
+Well darn&hellip; we can't find the page you're looking for. One of these links might help.
+
+* {% fa fa-angle-right %} {% url 'Introduction to Cypress' introduction-to-cypress %}
+* {% fa fa-angle-right %} {% url 'The API docs' api %}
+* {% fa fa-angle-right %} {% url 'Tutorials' tutorials %}
+* {% fa fa-angle-right %} {% url 'The Cypress Dashboard' https://www.cypress.io/dashboard/ %}
```

Comment:
I didn't add a link to the GitHub repository -- despite Gleb's requirement to do so -- because a link to the GitHub repo is in the footer.